modules/apps/data-engine/data-engine-taglib/test/js/data-layout-builder/components/rules/editor/Editor.es.js

```diff
@@ -371,19 +371,30 @@ describe('Editor', () => {
 							fireEvent.click(getByText('value'));
 						});
 
-						await waitFor(() => {
-							document
-								.querySelectorAll('.timeline-item')[1]
-								.querySelectorAll('.ddm-field')[3]
-								.querySelector(selector);
-						});
+						if (type === 'rich_text') {
+							await waitFor(() => {
+								document.querySelectorAll(selector);
+							});
```

Comment:
Hey @rebsilva, is this `waitFor` necessary? In my tests here it works without using it. :thinking:

Reply:
when i tested before it didn't work without it, i will try again :thinking:

Reply:
weird, could you test again? If it fails again you can keep waitFor and do the same as the comment below, move the `expect` into the `waitFor`.

Reply:
i've removed it!